tests/unit/test_client.py

```diff
@@ -12,11 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
 
 import mock
 from google.cloud.spanner_v1 import DirectedReadOptions
 
+hasOtelInstalled = False
+
+try:
+    from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+    from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+        InMemorySpanExporter,
+    )
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.sampling import ALWAYS_ON
+    from opentelemetry import trace
+
+    hasOtelInstalled = True
+except ImportError:
+    pass
```

Comment:
Can we directly infer this from _helpers.py similar to https://github.com/googleapis/python-spanner/blob/b2c69d433a129a6313b83e9978a2b7bae73814cf/tests/unit/test__opentelemetry_tracing.py#L15